setup.py

```diff
@@ -58,7 +55,7 @@
         'numpy >= 1.6',
         'OpenFisca-Core >= 0.5.0',
         'OpenFisca-Parsers >= 0.5',
-        'PasteDeploy',
```

Comment:
Need to talk with team @openfisca about the difference between both, but I've never used PasteDeploy, and the only place it was used was in `test/common.py`.